### PR TITLE
Refs #22969: Add keep_current_* midware tests

### PR DESCRIPTION
**Pull request by akofink**
First middleware tests of foreman-tasks!

### CODE REVIEW

**Comment by iNecas** [comment]
Running tests in http://ci.theforeman.org/job/test_plugin_pull_request/4823/

**Comment by akofink** [comment]
Updated with rubocop fixes.

**Comment by iNecas** [comment]
Rebuilding in http://ci.theforeman.org/job/test_plugin_pull_request/4825/

**Comment by iNecas** [comment]
Test failures unrelated, thanks @akofink

File: test/lib/actions/middleware/keep_current_user_test.rb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      describe 'run' do
+        before do
+          User.stubs(:current).returns(@user)
+
+          @action = create_and_plan_action(TestAction)
+
+          User.stubs(:current)
+        end
+
+        test 'with current taxonomies as input' do
```
Comment (akofink) [inline test/lib/actions/middleware/keep_current_user_test.rb]: Uh.. whoops, this one should be s/taxonomies/user/

Comment (iNecas) [reply test/lib/actions/middleware/keep_current_user_test.rb]: Could you send a small PR to update this?

Comment (akofink) [reply test/lib/actions/middleware/keep_current_user_test.rb]: https://github.com/theforeman/foreman-tasks/pull/328

**Pull request merged**